### PR TITLE
Feature/add nino to claimant api

### PR DESCRIPTION
**Pull request by mikeSmith08**

### CODE REVIEW

File: swagger.yml [hunk 1]
```diff
@@ -72,6 +72,11 @@ definitions:
         description: "Last (family) name"
         minLength: 1
         maxLength: 500
+      nino:
+        type: "string"
+        example: "QQ123456C"
+        description: "National Insurance number"
+        pattern: "[a-zA-Z]{2}\\d{6}[a-zA-Z]"
```
[the remark this answers is not in the document]
Comment (mikeSmith08) [reply swagger.yml]: Done.

File: api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/ClaimantTestDataFactory.java [hunk 2]
```diff
@@ -54,11 +52,7 @@ public static Claimant aValidClaimant() {
      * @return The built Claimant
      */
     public static Claimant aClaimantWithLastName(String lastName) {
-        return Claimant.builder()
-                .firstName("Joe")
-                .lastName(lastName)
-                .nino(VALID_NINO)
-                .build();
+        return buildClaimant(VALID_FIRST_NAME, lastName, VALID_NINO);
```
Comment (YetAnotherMatt) [inline api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/ClaimantTestDataFactory.java]: Not worth changing right now, but I'd suggest:
```
public static Claimant aClaimantWithLastName(String lastName) {
    return aValidClaimantBuilder().lastName(lastName).build;
}
```

File: swagger.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (YetAnotherMatt) [reply swagger.yml]: Need to re-run the build to regenerate swagger though.

File: api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/ClaimantTestDataFactory.java (hunk 2 above)
Comment (mikeSmith08) [reply api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/ClaimantTestDataFactory.java]: Yes, good suggestion, done.

File: swagger.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (mikeSmith08) [reply swagger.yml]: Done, swagger has been added (didn't spot the file to be committed in time).

**Pull request merged**